Lib/fontTools/ttLib/tables/TupleVariation.py

```diff
@@ -54,10 +55,7 @@ def hasImpact(self):
 		If the result is False, the TupleVariation can be omitted from the font
 		without making any visible difference.
 		"""
-		for c in self.coordinates:
-			if c is not None:
-				return True
-		return False
+		return any(c is not None for c in self.coordinates)
```

Comment:
if you had a function for first active delta you could use it here and in getDeltaType. Currently more or less duplicates the loop over coordinates.

Reply:
I don't think it's worth to turn a one-liner into its own method. The generator expression `next((c for c in self.coordinates if c is not None), None)` is efficient and readable to me.